float64[] goal_position
float64 Tmax
---
bool success
string info